hawkular-agent-core/src/main/java/org/hawkular/agent/monitor/storage/AsyncInventoryStorage.java

```diff
@@ -201,6 +201,10 @@ private String getInventoryId(NamedObject no) {
             mb.unit(MetricUnit.valueOf(metricUnits.name()));
         }
         m.getProperties().forEach((k, v) -> mb.property(k, v.toString()));
+
+        // TODO remove these once h-metrics integration is removed
+        mb.property("hawkular.metric.type", m.getType().getMetricType().name());
+        mb.property("hawkular.metric.typeId", m.getType().getID().getIDString());
```

Comment:
That's what I wanted to check with you on irc: we don't want the typeId here, but the metric Id (the string like "MT~feed~resource~etc." I don't remember exactly the pattern)

Reply:
And so, the key would be "hawkular.metric.id"